impactlab_tools/utils/paralog.py

```diff
@@ -159,6 +171,9 @@ def write(self, message):
         self.terminal.write(message)
         self.log.write(message)
 
+    def log_only(self, message):
```

Comment:
what is the benefit of having this outside `log_message`?

Reply:
This keeps the abstraction barrier around `DoubleLogger`, who is the only class that knows how to write to this log file.